weave-js/src/components/PagePanelComponents/Home/Browse3/inference/DropdownSelectSort.tsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * Sort order selector for the model catalog page.
+ */
+import {Button} from '@wandb/weave/components/Button';
+import * as DropdownMenu from '@wandb/weave/components/DropdownMenu';
+import React from 'react';
+
+import {Icon} from '../../../../Icon';
+
+export type ModelTileSortOrder =
+  | 'Popularity'
+  | 'Newest'
+  | 'Input price (low to high)'
+  | 'Input price (high to low)'
+  | 'Output price (low to high)'
+  | 'Output price (high to low)'
+  | 'Biggest context window';
```

Comment:
tioli: `Context window size`

Reply:
I matched what design specified but we'll definitely be doing a pass considering feedback like this.